Inputs/GraphQL/gqlQueries.robot

```diff
@@ -25,11 +25,9 @@ getRackSensorPointsOfGroupQuery
     ${query}=  set variable  query rackSensorPoints {site {groups: children(selector: {type: Group,name: "${group_name}"}) @skip(if:false) {oid name racks: children(selector:{type: Rack},){oid displayName points: children{oid name type pointCurrent{value}}}}}}
     return from keyword    ${query}
 
-# Query queryToFetchJsonResponseForSpecificAlarmType
-    #Created by Greeshma on 19 Aug 2021
 getAlarmStatusQuery
     [Arguments]   ${group_name}    ${alarm_name}
-    ${getAlarmStatusOfGroupQuery}=  set variable  query alarmStatus{ alarms(selector:{subjectName: "${group_name}", type : ${alarm_name}}) { type severity status }}
+    ${getAlarmStatusOfGroupQuery}=  set variable  query alarmStatus{ alarms(selector:{subjectName: "${group_name}", type : ${alarm_name}}) { type severity status message }}
```

Comment:
Instead of group_name, this variable can be named as component_name / subject_name as this query is applicable for group and ahus

Reply:
updated component_name  and added the comments that were deleted